packages/syft/src/syft/service/code/user_code.py

```diff
@@ -1065,7 +1066,12 @@ def to_str(arg: Any) -> str:
         exec(code_item.parsed_code, _globals, locals())  # nosec
 
         evil_string = f"{code_item.unique_func_name}(**kwargs)"
-        result = eval(evil_string, _globals, _locals)  # nosec
+        try:
```

Comment:
I think we should add a couple of things here
- make sure that the error is printed to stdout such that we can see it in the logs of the server side
- add the stacktrace to the error using traceback.format_exc() as in line 1096
- make sure that we add the error to the job, but in such a way that admins can see the error and users cannot. Later we can add some kind of permission on top of that such that its possible for users to request the error message.